gazelle/js/resolve.go

```diff
@@ -166,21 +166,27 @@ func (ts *TypeScript) resolveModuleDeps(
 			return nil, err
 		}
 
-		if resolutionType == Resolution_NotFound {
+		if resolutionType == Resolution_NotFound && cfg.ValidateImportStatements() != ValidationOff {
 			BazelLog.Debugf("import '%s' for target '%s' not found", mod.ImportPath, from.String())
 
-			if cfg.ValidateImportStatements() {
+			switch cfg.ValidateImportStatements() {
+			case ValidationError:
 				notFound := fmt.Errorf(
 					"Import %[1]q from %[2]q is an unknown dependency. Possible solutions:\n"+
 						"\t1. Instruct Gazelle to resolve to a known dependency using a directive:\n"+
 						"\t\t# gazelle:resolve [src-lang] js import-string label\n"+
 						"\t\t   or\n"+
 						"\t\t# gazelle:js_resolve import-string-glob label\n"+
 						"\t2. Ignore the dependency using the '# gazelle:%[3]s %[1]s' directive.\n"+
-						"\t3. Disable Gazelle resolution validation using '# gazelle:%[4]s false'",
+						"\t3. Disable Gazelle resolution validation using '# gazelle:%[4]s off'",
 					mod.ImportPath, mod.SourcePath, Directive_IgnoreImports, Directive_ValidateImportStatements,
 				)
-
+				resolutionErrors = append(resolutionErrors, notFound)
+			case ValidationWarn:
+				notFound := fmt.Errorf(
+					"Import %[1]q from %[2]q is an unknown dependency",
```

Comment:
Why not use the same message as the error case? The user might still need to fix it or want to ignore/resolve it to not see the message again...

Reply:
Fair point... it just seemed verbose for a warning, but consistency is good too